tests/run-make/target-specs-filename/rmake.rs

```diff
@@ -0,0 +1,25 @@
+// This test checks the rustc can accept target-spec-json with cases
+// including file extension - it could be in upper and lower case.
+// Used to test: print target-spec-json and cfg
+// Issue: https://github.com/rust-lang/rust/issues/127387
+
+///@ ignore-cross-compile
```

Comment:
@tgross35 ☝🏻 I'm not sure should it be here or I can remove ignore-cross-compile?

Reply:
Since you don't try to run a built binary, this should be able to handle cross-compilation. But we don't have to assume, we can run try-jobs to check!

Reply:
For context, I'm running your test on 4 CI runners: `test-various` and `armhf-gnu` CI runners in particular has cross-compilation setups that will exercise your test in cross-compilation context.

Reply:
Thank you, that's cool!

Reply:
As you said, [all works](https://github.com/rust-lang/rust/pull/127389#issuecomment-2211836260) without `ignore-cross-compile`. Bors competed test job 🎉